### ISSUE TITLE
Change storybook-vue to @storybook/vue

### ISSUE THREAD

**Issue by DrSensor** [egoist/poi#319]
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
when tried to init storybook using `getstorybook`, storybook will attempt to install `@storybook/vue` instead `storybook-vue`. It would be nice to depend on `@storybook/vue` instead `storybook-vue`

**If the current behavior is a bug, please provide the steps to reproduce.**
I found it when try to execute `getstorybook --force` in storybook-vue-example then have this error
```
Error: You have to install either storybook-vue or storybook-react!
    at poi.extendWebpack.config (/home/wildan/Projects/OSS/vue-annotator/node_modules/poi-preset-storybook/lib/index.js:50:15)
    at webpackFlows.forEach.flow (/home/wildan/Projects/OSS/vue-annotator/node_modules/poi/lib/index.js:77:39)
    at Array.forEach (<anonymous>)
    at Poi.createWebpackConfig (/home/wildan/Projects/OSS/vue-annotator/node_modules/poi/lib/index.js:77:23)
    at Promise (/home/wildan/Projects/OSS/vue-annotator/node_modules/poi/lib/index.js:187:34)
    at new Promise (<anonymous>)
    at Poi.runMiddlewares (/home/wildan/Projects/OSS/vue-annotator/node_modules/poi/lib/index.js:186:12)
    at prepare.then (/home/wildan/Projects/OSS/vue-annotator/node_modules/poi/lib/index.js:115:24)
    at <anonymous>
```

**What is the expected behavior?**
In `package.json`, changing `storybook-vue` to `@storybook/vue` then exec 
`poi --config .storybook/poi.config.js --storybook` should be okay.

**If this is a feature request, what is the motivation or use case for changing the behavior?**
It would be nice to depend on official library instead.

**Please mention other relevant information such as the browser version, Node.js version, Poi version and Operating System.**
Poi 9.4.3
poi-preset-storybook 0.1.0

**Comment by egoist** [comment]
Maybe we can remove that warning, but `@storybook/x` includes dependencies for webpack which are unnecessary for us 😄

**Comment by egoist** [comment]
Fixed in `poi-preset-storybook@0.1.3`
